validator/rpc/wallet_test.go

```diff
@@ -83,34 +83,55 @@ func TestServer_CreateWallet_Imported(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func TestServer_CreateWallet_Derived(t *testing.T) {
+func TestServer_RecoverWallet_Derived(t *testing.T) {
 	localWalletDir := setupWalletDir(t)
 	defaultWalletPath = localWalletDir
 	ctx := context.Background()
 	s := &Server{
 		walletInitializedFeed: new(event.Feed),
 		walletDir:             localWalletDir,
 	}
-	req := &pb.CreateWalletRequest{
-		Keymanager:     pb.KeymanagerKind_DERIVED,
+	req := &pb.RecoverWalletRequest{
 		WalletPassword: strongPass,
 		NumAccounts:    0,
 	}
-	// We delete the directory at defaultWalletPath as CreateWallet will return an error if it tries to create a wallet
+	// We delete the directory at defaultWalletPath as RecoverWallet will return an error if it tries to create a wallet
 	// where a directory already exists
 	require.NoError(t, os.RemoveAll(defaultWalletPath))
-	_, err := s.CreateWallet(ctx, req)
+	_, err := s.RecoverWallet(ctx, req)
 	require.ErrorContains(t, "Must create at least 1 validator account", err)
 
 	req.NumAccounts = 2
-	_, err = s.CreateWallet(ctx, req)
-	require.ErrorContains(t, "Must include mnemonic", err)
+	_, err = s.RecoverWallet(ctx, req)
+	require.ErrorContains(t, "input not in the list of allowed languages", err)
+
+	req.Language = "english"
+	_, err = s.RecoverWallet(ctx, req)
+	require.ErrorContains(t, "invalid mnemonic in request", err)
 
 	mnemonicResp, err := s.GenerateMnemonic(ctx, &empty.Empty{})
 	require.NoError(t, err)
 	req.Mnemonic = mnemonicResp.Mnemonic
 
-	_, err = s.CreateWallet(ctx, req)
+	req.Mnemonic25ThWord = " "
+	req.SkipMnemonic_25ThWord = false
```

Comment:
```suggestion
	req.SkipMnemonic25thWord = false
```

Reply:
Ok.  So for SkipMnemonic25thWord  , it has to be defined in proto as skip_mnemonic_25th_word.   I'll do like we did with mnemonic_25th_word up .